jenkins/check-for-build.jenkinsfile

```diff
@@ -7,6 +7,7 @@ pipeline {
     agent none
     triggers {
         parameterizedCron '''
+            H/10 * * * * %INPUT_MANIFEST=1.2.5/opensearch-1.2.5.yml;TARGET_JOB_NAME=distribution-build-opensearch
             H/10 * * * * %INPUT_MANIFEST=1.3.0/opensearch-dashboards-1.3.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards
             H/10 * * * * %INPUT_MANIFEST=1.2.4/opensearch-1.2.4.yml;TEST_MANIFEST=1.2.4/opensearch-1.2.4-test.yml;TARGET_JOB_NAME=distribution-build-opensearch
```

Comment:
@abhinavGupta16 Can you remove this 1.2.4? Thanks

Reply:
We do not need to remove this as it will not trigger since we are locked to tags already.

Reply:
Tho I think @dblock original design is not having a function to detect whether a manifest is done and deal, so no rebuild is needed anymore.

Reply:
Its a cronjob and 1.2.4 is already out. We remove the released version from here as there is no benefit in keeping it here?

Reply:
https://github.com/opensearch-project/opensearch-build/pull/1596

Reply:
This does not work because the auto workflow will add it back tomorrow.

Reply:
It only adds for new version which is `1.2.5`

Reply:
Verified thanks.
Approved.